en/admin/admin_utils.rst

```diff
@@ -2326,6 +2339,30 @@ The following shows [options] available with the **cubrid statdump** utility.
     | Num_lfcq_shared_lists                    | Snapshot       | The current number of non-zero candidate shared LRU lists             |
     +------------------------------------------+----------------+-----------------------------------------------------------------------+
 
+    **Double write buffer**
+
+    +------------------------------------------+----------------+-----------------------------------------------------------------------+
+    | **Stat name**                            | **Stat type**  |  **Description**                                                      |
+    +------------------------------------------+----------------+-----------------------------------------------------------------------+
+    | ..Time_DWB_flush_block_time              | Counter/timer  | The number and duration of DWB block flushes                          |
+    +------------------------------------------+----------------+-----------------------------------------------------------------------+
+    | ..Time_DWB_flush_block_helper_time       | Counter/timer  | ??                                                                    |
+    +------------------------------------------+----------------+-----------------------------------------------------------------------+
+    | ..Time_DWB_flush_block_cond_wait_time    | Counter/timer  | The number and duration of DWB thread sleeps                          |
+    +------------------------------------------+----------------+-----------------------------------------------------------------------+
+    | ..Time_DWB_flush_block_sort_time         | Counter/timer  | The number and duration of sorting pages for flush                    |
+    +------------------------------------------+----------------+-----------------------------------------------------------------------+
+    | ..Time_DWB_flush_remove_hash_entries     | Counter/timer  | The number and duration of removing page from DWB                     |
+    +------------------------------------------+----------------+-----------------------------------------------------------------------+
+    | ..Time_DWB_checksum_time                 | Counter/timer  | !!Unused                                                              |
+    +------------------------------------------+----------------+-----------------------------------------------------------------------+
+    | ..Time_DWB_wait_flush_block_time         | Counter/timer  | ??                                                                    |
+    +------------------------------------------+----------------+-----------------------------------------------------------------------+
+    | ..Time_DWB_wait_flush_block_helper_time  | Counter/timer  | ??                                                                    |
```

Comment:
flush block daemon must waits for previous block to be completely flushed, before flushing the current block.